src/modality-specific-files/magnetoencephalography.md

```diff
@@ -98,8 +98,10 @@ it SHOULD be stored separately under a new `/eeg` data type
 If however EEG is recorded simultaneously **with the same MEG system**,
 it MAY be stored under the `/meg` data type.
 In that case, it SHOULD have the same sampling frequency as MEG (see `SamplingFrequency` field below).
-Furthermore, the EEG sensor coordinates SHOULD be specified using MEG-specific coordinate
-systems (see [coordinates section](#coordinate-system-json-_coordsystemjson) below and
+Furthermore, EEG sensor coordinates SHOULD be recorded in an
```

Comment:
IMHO the SHOULD should become a MAY.

So far, the spec allowed for recording electrode positions in the `_headshape`: https://bids-specification.readthedocs.io/en/stable/glossary.html#headshape-suffixes